File: beecrowd/C++/1009.cpp
#include <iostream>
#include <iomanip>
using namespace std;

int main()
{
    string nome;
    cin >> nome;

    double salarioFixo;
    cin >> salarioFixo;

    double vendas;
    cin >> vendas;

    double comissao = vendas * 0.15;
    double salarioMes = comissao + salarioFixo;

    cout << fixed << setprecision(2);
    cout << "TOTAL = R$ " << salarioMes << endl;

    return 0;
} // end main

File: beecrowd/Java/1009.java
import java.io.IOException;
import java.util.Scanner;

public class Main{
    public static void main(String [] args) throws IOException{
        Scanner keyboard = new Scanner (System.in);

        String nome = keyboard.nextLine();
        double salarioFixo = keyboard.nextDouble();
        double vendasMes =  keyboard.nextDouble();

        double bonificacao = vendasMes * 0.15;
        double salarioMes = salarioFixo + bonificacao;

        System.out.printf("TOTAL = R$ %.2f\n", salarioMes);
        keyboard.close();
    }
}